src/style.scss

```diff
@@ -3,118 +3,142 @@
 
 // ---------------------------------------------------------------------
 
-@import './node_modules/@alaskaairux/orion-design-tokens/dist/tokens/SCSSVariables';
-@import './node_modules/@alaskaairux/orion-design-tokens/dist/tokens/SassCustomProperties';
+/* stylelint-disable declaration-empty-line-before, max-nesting-depth, selector-max-class, selector-class-pattern, selector-max-combinators, selector-max-compound-selectors, no-descending-specificity, scss/double-slash-comment-empty-line-before, order/properties-order */
 
-@import './node_modules/@alaskaairux/orion-web-core-style-sheets/dist/breakpoints';
-@import './node_modules/@alaskaairux/orion-web-core-style-sheets/dist/core';
-@import './node_modules/@alaskaairux/orion-web-core-style-sheets/dist/animation';
+@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
+@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties';
+
+@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/breakpoints';
+@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/core';
+@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/animation';
 
 :host {
   display: inline-block;
 }
 
-@mixin sm{
+@mixin sm {
   .pane {
-    @include auro_transition(all, 0.15s, ease);
-
+    @include auro_transition(all, .15s, ease);
     // fallback for browsers that don't support grid
     // also Safari, which does not support display: grid on buttons
-    display: flex; 
-    flex-direction: column;
+
+    display: flex;
+    min-width: 4.25rem;
+    height: 5.625rem;
+
     align-items: center;
+    flex-direction: column;
 
-    @supports (display: grid) {
-      display: grid;
-      grid-template-columns: auto;
-      grid-template-rows: 1.625rem 1.25rem;
-    }
-    
     padding-top: 1.375rem;
-    height: 5.625rem;
-    min-width: 4.25rem;
-    font-family: var(--auro-font-family-default);
-    font-weight: var(--auro-text-body-default-weight);
 
-    background: var(--auro-color-base-gray-100);
     border: 2px solid;
     border-color: var(--auro-color-base-gray-200);
     border-radius: 2.5rem;
+
+    background: var(--auro-color-base-gray-100);
+
     cursor: pointer;
 
+    font-family: var(--auro-font-family-default);
+    font-weight: var(--auro-text-body-default-weight);
+
+    @supports (display: grid) {
+      display: grid;
+
+      grid-template-columns: auto;
+      grid-template-rows: 1.625rem 1.25rem;
+    }
+
     @media (hover: hover) {
       &:hover {
-        background: var(--auro-color-border-active-on-light);
         border-color: var(--auro-color-border-active-on-light);
 
+        background: var(--auro-color-border-active-on-light);
```

Comment:
With the latest design token release, the `-on-light` tokens were deprecated. Might be just a good opportunity here to update them now. 

https://auro.alaskaair.com/getting-started/developers/design-tokens/deprecated